__tests__/park.test.js

```diff
@@ -1,105 +1,205 @@
+      .put({
+        TableName: TABLE_NAME,
+        Item: park,
+      })
+      .promise();
   }
-}
 
-async function genericPutDocument(item) {
-  return await docClient
-    .put({
-      TableName: TABLE_NAME,
-      Item: item
-    })
-    .promise();
+  for (const subarea of SUBAREAS) {
+    await docClient
+      .put({
+        TableName: TABLE_NAME,
+        Item: subarea,
+      })
+      .promise();
+  }
 }
 
-describe('Fiscal Year End Test', () => {
+describe("Pass Succeeds", () => {
   beforeAll(async () => {
     return await setupDb();
   });
 
-  test('Handler - 200 GET fiscal year end', async () => {
-    const obj = await fiscalYearEndGET.handler(
+  test("Handler - 200 Received list of parks", async () => {
+    const event = {
+      headers: {
+        Authorization: "Bearer " + token,
+      },
+      httpMethod: "GET",
+    };
+    expect(await parkGET.handler(event, null)).toMatchObject({
+      body: JSON.stringify(PARKSLIST),
+      headers: {
+        "Access-Control-Allow-Headers":
+          "Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token",
+        "Access-Control-Allow-Methods": "OPTIONS,GET,POST",
+        "Access-Control-Allow-Origin": "*",
+        "Content-Type": "application/json",
+      },
+      statusCode: 200,
+    });
+  });
+
+  test("Handler - 200 Receive list of parks with limited role", async () => {
+    let specificSubAreas = [];
+    for (const area of SUBAREAS) {
+      if (area.pk === "park::0041") {
+        specificSubAreas.push(area);
+      }
+    }
+    const response = await parkGET.handler(
       {
         headers: {
-          Authorization: "Bearer " + token
+          Authorization: "Bearer " + token,
+          PsuedoToken: roleToken,
         },
-        queryStringParameters: {
-          fiscalYearEnd: FISCAL_YEAR_LOCKS[0].sk
-        }
-      }, null);
-    expect(JSON.parse(obj.body)).toMatchObject(FISCAL_YEAR_LOCKS[0]);
+      },
+      null
+    );
+
+    const body = JSON.parse(response.body);
+    // Body should be empty
+    expect(body).toMatchObject([]);
   });
 
-  test('Handler - 200 GET All fiscal year end objects', async () => {
-    const response = await fiscalYearEndGET.handler({
-      headers: {
-        Authorization: "Bearer " + token
+  test("Handler - 200 Receive park specific information", async () => {
+    let specificSubAreas = [];
+    for (const area of SUBAREAS) {
+      if (area.pk === "park::0041") {
+        specificSubAreas.push(area);
+      }
+    }
+    const response = await parkGET.handler(
+      {
+        headers: {
+          Authorization: "Bearer " + token,
+        },
+        queryStringParameters: {
+          orcs: PARKSLIST[0].sk,
+          subAreaId: specificSubAreas[0].sk,
+        },
       },
-    }, null);
+      null
+    );
 
-    expect(response.statusCode).toBe(200);
+    const body = JSON.parse(response.body);
+    expect(body.data[0].subAreaName).toMatch(specificSubAreas[0].subAreaName);
```

Comment:
I think it's best to leave the expect `200` in there, with the additional check on object matching.